packages/arb-token-bridge-ui/src/util/WithdrawOnlyUtils.ts

```diff
@@ -281,29 +282,119 @@ export const withdrawOnlyTokens: { [chainId: number]: WithdrawOnlyToken[] } = {
   ]
 }
 
-async function isLayerZeroToken(
+/**
+ * Fetches LayerZero's off-chain metadata (https://metadata.layerzero-api.com/v1/metadata) to identify OFT tokens.
+ * If found in the metadata, it means the token supports OFT transfers - hence shouldn't be deposited through Arbitrum's canonical bridge.
+ * @param parentChainErc20Address
+ * @param parentChainId
+ * @returns boolean - true if the token is an OFT token, false otherwise
+ */
+async function isLayerZeroTokenViaAPI(
   parentChainErc20Address: string,
   parentChainId: number
-) {
-  const parentProvider = getProviderForChainId(parentChainId)
+): Promise<boolean> {
+  const chainIdToLzName: Record<number, string | undefined> = {
+    [ChainId.Ethereum]: 'ethereum',
+    [ChainId.ArbitrumOne]: 'arbitrum',
+    [ChainId.ArbitrumNova]: 'nova',
+    [ChainId.Sepolia]: 'ethereum-sepolia',
+    [ChainId.ArbitrumSepolia]: 'arbitrum-sepolia'
+  }
+
+  const parentChainName = chainIdToLzName[parentChainId]
+
+  if (!parentChainName) {
+    // We can't check via the API if the chain is not supported
+    throw new Error(`No LayerZero chain name for chain ID ${parentChainId}`)
+  }
 
-  // https://github.com/LayerZero-Labs/LayerZero-v2/blob/592625b9e5967643853476445ffe0e777360b906/packages/layerzero-v2/evm/oapp/contracts/oft/OFT.sol#L37
-  const layerZeroTokenOftContract = new ethers.Contract(
-    parentChainErc20Address,
-    [
-      'function oftVersion() external pure virtual returns (bytes4 interfaceId, uint64 version)'
-    ],
-    parentProvider
+  // Fetches LayerZero's off-chain metadata to identify OFT tokens.
+  // If found in the metadata, it means the token supports OFT transfers - hence shouldn't be deposited through Arbitrum's canonical bridge.
+  // Schema:
+  // {
+  //   "ethereum": { <-- parent chain name
+  //     "tokens": {
+  //       "0x57e114b691db790c35207b2e685d4a43181e6061": {  <--- parent chain erc20 address
+  //         "id": "ena",
+  //         "symbol": "ENA",
+  //         "decimals": 18
+  //       }
+  //       ...more tokens
+  //     }
+  //   }
+  // }
+
+  const response = await axios.get(
```

Comment:
Any reason we use axios over fetch here? We don't seem to leverage any of the utilities of axios